Source/Core/Duality/Backend/Dummy/DummyGraphicsBackend.cs

```diff
@@ -30,7 +30,10 @@ bool IDualityBackend.CheckAvailable()
 		{
 			return true;
 		}
-		void IDualityBackend.Init() { }
+		void IDualityBackend.Init()
+		{
+			Log.Core.WriteWarning("DummyGraphicsBackend initialized. This is unusual and may cause problems when someone tries to load images.");
```

Comment:
Loading images (as in CPU side image data) is actually what the system backend does. For the graphics backend, please change to a hint regarding interaction with graphics devices and rendering.